microsoft_atp_safe_links/komand_microsoft_atp_safe_links/util/utils.py

```diff
@@ -8,4 +8,4 @@ def decode_url(url: str) -> str:
     for param in split_url_params:
         if param.split("=")[0] == "url":
             return unquote(param.split("=")[1])
-    return ""
+    return url
```

Comment:
I think we can do more here to ensure it's a URL...however, the exit case in the action just returns the input as well...so eh...I can go either way.